networkit/cpp/algebraic/test/MatricesGTest.cpp

```diff
@@ -828,6 +931,65 @@ TYPED_TEST(MatricesGTest, testMatrixMultiplication) {
     EXPECT_EQ(result(2, 1), 4);
 }
 
+TYPED_TEST(MatricesGTest, testMatricesTransposedMultiplication) {
+    using Matrix = typename TestFixture::Matrix;
+    if (std::is_same<Matrix, DenseMatrix>::value) {
+        GTEST_SKIP() << "Skipping mmT/mTm test for DenseMatrix";
+    }
+    std::vector<Triplet> triplets1 = {{0, 0, 1}, {0, 1, 2}, {0, 2, 3}, {1, 0, 2}, {1, 1, 2}};
+    //		     1  2  3
```

Comment:
Indentation looks broken.

Reply:
indentation looks as intended in VScode (displaying the matrices/vectors visually)